chainstate/test-suite/src/tests/pos_accounting_reorg.rs

```diff
@@ -85,7 +85,7 @@ fn stake_pool_reorg(#[case] seed: Seed) {
                 .add_output(TxOutput::CreateStakePool(
                     pool_id_a,
                     Box::new(StakePoolData::new(
-                        Amount::from_atoms(rng.gen_range(100_000..200_000)),
+                        min_pledge,
```

Comment:
Please preserve having a range... this helps in randomizing things and detecting potential issues.

Reply:
This doesn't apply to this only, to be clear.